ndcube/ndcube.py

```diff
@@ -641,3 +642,37 @@ def __init__(self, data, wcs, uncertainty=None, mask=None, meta=None,
                          mask=result_mask, meta=meta, unit=unit,
                          extra_coords=reordered_extra_coords,
                          copy=copy, missing_axis=missing_axis, **kwargs)
+
+
+def fits_ndcube_writer(ndcube_data, filename, hdu_mask='MASK', hdu_uncertainty='UNCERT',  key_uncertainty_type='UTYPE', **kwargs):
```

Comment:
I think this should be moved to the utils/cube.py module.

Reply:
Sure